service/service_test.go

```diff
@@ -135,10 +107,172 @@ func TestNew(t *testing.T) {
 		mock.Anything,
 		mock.Anything,
 	).Return(false, nil)
+	config, store := validConfig(t)
+	lc := newLotusClientMock()
 	_, err = service.New(config, store, lc, fc2)
 	require.Error(t, err)
 }
 
+func TestBytesLimit(t *testing.T) {
```

Comment:
might be worth trying a newer badger release. unless we updated since the start of bidbot, there have been a few releases for v3... latest release: https://github.com/dgraph-io/badger/releases/tag/v3.2103.1